samples/gen-go-db/server/handlers.go

```diff
@@ -0,0 +1,123 @@
+package server
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"github.com/Clever/wag/samples/gen-go-db/models"
+	"github.com/go-errors/errors"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/gorilla/mux"
+	"gopkg.in/Clever/kayvee-go.v6/logger"
+)
+
```

Comment:
I'm familiar with this syntax.

In the other file you have a comment `// These imports may not be used depending on the input parameters"`

=> Still not sure I understand it.
=> Probably want to include a comment in all places you do this, if relevant